### DISCUSSION TITLE
PHPMailer relying on Interworx with Let's Encrypt SSL stopped working on September 30th

### DISCUSSION THREAD

**Discussion by BryceMorrison** [PHPMailer/PHPMailer#2518] [Q&A]
Up until September 30th around 4pm, the PHPMailer was working on a Linux server with the Interworx/Siteworx alternative to cPanel, and the SSL installation was the free, included SSL, which is provided by Let's Encrypt.

The SSL connections stopped working on September 30th, which is coincidentally the same date for which Let's Encrypt deprecated DST Root CA X3 and required the use of ISRG Root X1.

https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/

I did reissue an SSL certificate via Interworx, resetting the start date to today, but that did not solve the problem.

Although this might be related indirectly to PHPMailer, I thought it appropriate to post this problem, as anyone using the free Let's Encrypt for the SSL ($mail->SMTPSecure on PHPMailer) might have the same issue.

**Comment by Synchro** [comment]
Make sure you're talking about the correct certificate. It's the certificate presented by the *mail* server that you're connecting to that matters; it has nothing to do with any certificate that you are using for your web site.

**Reply by BryceMorrison** [reply]
Yes, the mail is hosted on the server, rather than using a provider such as Gmail.

**Reply by Synchro** [reply]
Ok, in that case the most likely culprit is the CA certificate bundle installed in your OS and used by PHP. Read the troubleshooting guide and check out projects like certainty to be sure you have an up to date bundle.
